src/Select/index.js

```diff
@@ -16,9 +16,10 @@ const { bool, func, node, string } = PropTypes;
  *
  * @param {node} children // Anything that can be rendered: numbers, strings, elements or an array (or fragment).
  * @param {string} className /// className needed by styled components.
- * @param {function} onSelected // Function fired when an element of the <Select /> is selected.
+ * @param {function} onChange // Function fired when an element of the <Select /> is selected.
  * @param {function} onToggle // Function fired when the <Select /> is toggled.
- * @param {string} title // Title of the <Select /> component.
+ * @param {string} placeholder // placeholder of the <Select /> component.
+ * @param {boolean} resetValue // Title of the <Select /> component.
```

Comment:
Why `resetValue` and not `title` ?

Reply:
@AntoninMarchardDev Because we want to reset the value of the select. When is no value instead a placeholder is showed